### PR TITLE
Write library description in the README

### LINKED ISSUE #3: Add a README
Proposal:

> Telemetry is a dynamic dispatching library for metrics and instrumentations. It is lightweight, small and can be used in any Elixir project.
>
> In a nutshell, you register a custom module and function to be invoked for certain events, which are executed whenever there is such event. Event names are a list of atoms. Each event is composed of a numeric value and can have metadata attached to it. Let's see an example:
>
> ...

### PR DESCRIPTION
**Pull request by arkgil**
I added content to the README based on @josevalim's suggestions in #3. I think that there is a lot of room for improvement 😅 

Closes #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,44 @@
...
+```
+
+Finally, in your application code you would run:
+
+```elixir
+Telemetry.execute([:web, :request, :done], latency, %{request_path: path, status_code: status})
+```
+
+You might think that it isn't very useful, because you could just as well write a log statement
+instead of `Telemtry.execute/2` - and you would be right! But now imagine that each Elixir library
```
Comment (josevalim) [inline README.md]: Typo on the module name!

**Comment by arkgil** [comment]
I guess it can be merged.

**Comment by josevalim** [comment]
@arkgil maybe I am missing something but why don't you merge it? :) In OSS we usually let the author merge their own PRs if they have access and it is approved.

Anyway, I am glad to make your wish come true! :D

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by arkgil** [comment]
@josevalim I'm just used to "not merge your own PRs" rule :D Anyway, now I know, thanks!

**Pull request merged**